tests/README.md

```diff
@@ -570,6 +570,29 @@ ipdb> test_microvm.help.gdbserver()
 
 You get some instructions on how to run GDB to attach to gdbserver.
 
+## How to run tests with a different version of Firecracker
+
+The integration tests usually compile Firecracker as part of the test
+initialization. But there's an option in case we want to run the tests against a
+different version of Firecracker, for example a previous release:
+
+```sh
+./tools/devtool test -- --binary-dir ../v1.4.1
+```
+
+## How to run tests outside of Docker
+
+Tested in Ubuntu 22.04, AL2023. AL2 does not work due to an old Python (3.8).
+
+```sh
+sudo yum install python3-pip
```

Comment:
Mh, above we say its tested for ubuntu and AL2023, but for ubuntu this should be `apt-get`. Maybe default to `apt-get` here (as an open source contributor is more likely to be using some debian derivative than amazon linux) with a note to replace `apt-get` with `yum` for other distros? Or maybe we can just assume a working python installation?